index.js

```diff
@@ -1,4 +1,48 @@
 
+//removes grid container from document flow
+document.getElementById('gridContainer').style.display="none"
+
+//function to generate all game tiles
+document.querySelector('.form').addEventListener('submit', (e) => {
+    //Defining total grid size, number of lynx and misses from user input
+    let numLynx = document.getElementById('numLynx').value
+    let gridSize = document.getElementById('gridSize').value
+    let miss = gridSize - document.getElementById('numLynx').value
```

Comment:
could this be:
let miss = gridSize - numLynx 
As you are just typing out the same variable from line 8